doc/changelog.md

```diff
@@ -1,55 +1,100 @@
-# Release 0.3.2-dev
+# Release 0.3.2
 
 <h3>New features</h3>
```

Comment:
Just a reminder to include PR #347 changelog.

Reply:
done!